pdns/histog.hh

```diff
@@ -21,7 +21,7 @@ struct LogHistogramBin
 
 template<typename T>
 std::vector<LogHistogramBin> createLogHistogram(const T& bins,
-                        std::deque<double> percentiles={0.001, 0.01, 0.1, 0.2, 0.5, 1, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 94, 95, 96, 97, 97.5, 98, 98.5, 99, 99.5, 99.6, 99.9, 99.99, 99.999, 99.9999})
+                        std::deque<double>& percentiles={0.001, 0.01, 0.1, 0.2, 0.5, 1, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 94, 95, 96, 97, 97.5, 98, 98.5, 99, 99.5, 99.6, 99.9, 99.99, 99.999, 99.9999})
```

Comment:
I'm afraid binding a default value to a reference can't work, unless I'm mistaken. Note that this file was never actually compiled before https://github.com/PowerDNS/pdns/pull/7291.

Reply:
which should not work there :/